README.md

```diff
@@ -16,6 +16,22 @@ You can specify the desired template from the command line. This is useful for a
 npm create hono@latest ./my-app -- --template cloudflare-pages
 ```
 
+### `--install`
+
+Install dependencies after cloning template.
+
+```
+npm create hono@latest ./my-app -- --install
```

Comment:
I think `--` is not necessary. We can write it shorter.

```
npm create hono@latest ./my-app --install
```

Reply:
Unfortunately, it is necessary 😓 . A weird quirk of `npm create` is that it requires `--` to be specified before any additional flags that are intended to be passed to an underlying cli. This can be demonstrated with the existing `--template` flag:

![image](https://github.com/honojs/create-hono/assets/204386/998b9177-9a6c-4237-8d0a-a23fcd53d496)

`npm` is the only package manager with this weird quirk however. See pnpm:

![image](https://github.com/honojs/create-hono/assets/204386/aa418dfa-35ff-470a-b731-61d24d4480d0)

Reply:
@jculvey Thank you very very much for explaining the details. I could understand well. `--` should be needed!